pyrax/cloudblockstorage.py

```diff
@@ -283,7 +283,7 @@ def _create_body(self, name, size=None, volume_type=None, description=None,
         """
         try:
             int(size)
-        except:
+        except ValueError:
```

Comment:
Looking at this further (even though I changed it in a PR you already merged), I think it's better to do a hard check against six's integer types because:

```python
>>> int(2.75)
2
>>> from decimal import Decimal
>>> int(Decimal('2.75'))
2
```

Reply:
ah okay